bin/lint.ml

```diff
@@ -11,10 +11,12 @@ let check () =
 
 let main () =
   match check () with
-  | Ok () -> ()
-  | Error (`Msg msg) -> Stdio.print_endline msg
+  | Ok () -> 0
```

Comment:
You can make commands return an `(int, Rresul.R.msg) result` if you want to keep the ability to fail without having to return an `Error` which can be convenient for linters!

Reply:
I'm assuming I'll change the structure to some degree earlier than later so this can be incorporated at that point.